package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "mocha-chrome",
-  "version": "2.2.0",
+  "name": "digabi-mocha-chrome",
```

Comment:
I cant accept this change. I wouldn't recommend having your team commit to a fork branch that has an open PR